devito/interfaces.py

```diff
@@ -1,9 +1,47 @@
 import numpy as np
 from sympy import IndexedBase
 from tools import aligned
+from hashlib import sha1
+from random import randint
+import os
 
 
 class DenseData(IndexedBase):
+
+    # hash function used for generating part of memmap file name
+    _hashing_function = sha1
+    # this is the directory where memmap files will be created
+    _disk_path = None
```

Comment:
Not sure this should be a class attribute. It is conceivable that a user might want different output files in different directories, no?